docs/src/content/docs/es/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,291 @@
+
+    /** @type {import('tailwindcss').Config} */
+    module.exports = {
+    	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    	plugins: [starlightPlugin()],
+    };
+    ```
+
+### Estilando Starlight con Tailwind
+
+Starlight utilizará los valores de su [configuración de tema Tailwind](https://tailwindcss.com/docs/theme) en su UI.
+
+Si se establece, las siguientes opciones anularán los estilos predeterminados de Starlight:
+
+- `colors.accent` - usada para enlaces y resaltado de elementos actuales
+- `colors.gray` - usada para colores de fondo y bordes
+- `fontFamily.sans` - usada para el texto de la UI y el contenido
+- `fontFamily.mono` - usada para ejemplos de código
+
+```js {12,14,18,20}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Tu color de acento preferido. Indigo es el más cercano a los valores predeterminados de Starlight.
+				accent: colors.indigo,
+				// Tu escala de grises preferida. Zinc es el más cercano a los valores predeterminados de Starlight.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				// Tu fuente de texto preferida. Starlight usa una pila de fuentes del sistema de forma predeterminada.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Tu fuente de código preferida. Starlight usa fuentes monoespaciadas del sistema de forma predeterminada.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Temas
+
+El tema de color de Starlight se puede controlar anulando sus propiedades personalizadas predeterminadas.
+Estas variables son utilizadas en toda la UI con una gama de tonos grises utilizados para los colores de texto y fondo y un color de acento utilizado para los enlaces y para resaltar los elementos actuales en la navegación.
+
+### Editor de temas de color
+
+Usa los controles deslizantes a continuación para modificar las paletas de colores de acento y gris de Starlight.
+Las áreas de vista previa oscura y clara mostrarán los colores resultantes, y toda la página también se actualizará para obtener una vista previa de sus cambios.
+
+Cuando estés satisfecho con tus cambios, copia el código CSS o Tailwind a continuación y úsalo en tu proyecto.
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Presets',
+			ocean: 'Ocean',
+			forest: 'Forest',
+			oxide: 'Oxide',
+			nebula: 'Nebula',
+			default: 'Default',
+			random: 'Random',
+		},
+		editor: {
+			accentColor: 'Accent',
+			grayColor: 'Gray',
+			hue: 'Hue',
+			chroma: 'Chroma',
+			pickColor: 'Pick color',
+		},
+		preview: {
+			darkMode: 'Dark mode',
+			lightMode: 'Light mode',
```

Comment:
Oh, sorry, I think my last review was written confusingly. These labels can also be translated!

Something like this?

```suggestion
			label: 'Preajustes',
			ocean: 'Océano',
			forest: 'Bosque',
			oxide: 'Óxido',
			nebula: 'Nebulosa',
			default: 'Por defecto',
			random: 'Aleatorio',
		},
		editor: {
			accentColor: 'Acento',
			grayColor: 'Gris',
			hue: 'Tono',
			chroma: 'Croma',
			pickColor: 'Elegir color',
		},
		preview: {
			darkMode: 'Tema oscuro',
			lightMode: 'Tema claro',
```

Otherwise this all looks good! 🙌